lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_qatar.govspeak.erb

```diff
@@ -7,321 +7,55 @@
   <% if calculator.resident_of_uk? %>
     <%= render partial: 'contact_embassy_of_ceremony_country_in_uk_marriage.govspeak.erb',
                locals: { calculator: calculator } %>
-  <% elsif calculator.resident_of_ceremony_country? || calculator.ceremony_country == 'qatar' %>
+  <% else %>
     <%= render partial: 'contact_local_authorities_in_country_marriage.govspeak.erb',
                locals: { calculator: calculator } %>
-    <% if calculator.ceremony_country == 'qatar' %>
-      <%= render partial: 'gulf_states_os_consular_cni.govspeak.erb',
-                 locals: { calculator: calculator } %>
-      <%= render partial: 'gulf_states_os_consular_cni_local_resident.govspeak.erb',
-                 locals: { calculator: calculator } %>
-
-    <% end %>
-  <% elsif calculator.resident_of_third_country? %>
-    <%= render partial: 'contact_local_authorities_in_country_marriage.govspeak.erb',
+    <%= render partial: 'gulf_states_os_consular_cni.govspeak.erb',
                locals: { calculator: calculator } %>
-
-  <% elsif calculator.resident_of_ceremony_country && calculator.ceremony_country == 'thailand' %>
-    <%= render partial: 'contact_local_authorities_in_country_marriage.govspeak.erb',
+    <%= render partial: 'gulf_states_os_consular_cni_local_resident.govspeak.erb',
                locals: { calculator: calculator } %>
 
   <% end %>
 
 
-  <% if %w(ecuador).exclude?(calculator.ceremony_country) %>
-    <% if calculator.resident_of_ceremony_country? %>
-      <%= render partial: 'get_legal_advice.govspeak.erb' %>
-    <% else %>
-      <%= render partial: 'get_legal_and_travel_advice.govspeak.erb',
-                 locals: { calculator: calculator } %>
-    <% end %>
-  <% end %>
-
-  <% if calculator.ceremony_country == 'thailand' %>
-    %You must register your marriage at an amphur office (district office) in Thailand - a religious ceremony on its own won’t be valid.%
-  <% end %>
-
-  <% if calculator.ceremony_country == 'united-arab-emirates' %>
-    ^There are no civil marriages in the United Arab Emirates, but you can get married through a religious ceremony at a church or Sharia court.^
-
-    You can only get married in the United Arab Emirates if you or your partner are resident there.
-  <% end %>
-
-  <% if %w(turkey egypt china).include?(calculator.ceremony_country) %>
-    <%= render partial: 'what_you_need_to_do.govspeak.erb',
-        locals: { calculator: calculator } %>
-  <% elsif calculator.opposite_sex_21_days_residency_required? %>
```

Comment:
Minor: It's hard to see this in the diff without the `--ignore-all-spaces` option enabled, but some "What you need to do" content related to the `opposite_sex_21_days_residency_required` is retained in this commit. It wasn't immediately obvious to me why this was the case. Having investigated it a bit, I see that `qatar` is in the list of `OS_21_DAYS_RESIDENCY_REQUIRED_COUNTRIES` in the Marriage Abroad data query class. I think it would be worth mentioning this in the commit note.